.github/workflows/python-build-package.yml

```
@@ -76,35 +76,27 @@ jobs:
       - run: "python3 -c 'import magika; m = magika.Magika(); print(m)'"
       - run: magika -r tests_data/basic
       - run: python3 ./python/scripts/run_quick_test_magika_cli.py
-        # Windows' onnxruntime/ort returns different results wrt. Linux/MacOS. For
-        # now keep building even in case of misdetections.
-        # TODO(https://github.com/google/magika/issues/892): remove when this is fixed.
-        continue-on-error: ${{ matrix.platform.runner == 'windows-latest' }}
       - run: python3 ./python/scripts/run_quick_test_magika_module.py
-        # TODO(https://github.com/google/magika/issues/892): same as above,
-        # remove when this is fixed.
-        continue-on-error: ${{ matrix.platform.runner == 'windows-latest' }}
       - name: Upload wheels
         if: github.event_name != 'pull_request'
         uses: actions/upload-artifact@v4
         with:
           name: wheels-${{ matrix.platform.runner }}-${{ matrix.platform.target }}
           path: dist
 
-  sdist:
+  build-pure-python-wheel:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - run: python3 ./python/scripts/fix_package_version.py
-      - name: Build sdist
-        uses: PyO3/maturin-action@v1
+      - run: uv run ./scripts/prepare_pyproject_for_pure_python_wheel.py
+          working-directory: python
```

I would expect less indentation here.
```suggestion
        working-directory: python
```

---

👍